setup.py

```diff
@@ -23,6 +23,7 @@
     'absl-py',
     'attrs',
     'chex',
+    'h5py==2.10.0',
```

Comment:
Please add a TODO comment to remote this restriction once pyscf 2.0 is on pypi. (See https://github.com/google/styleguide/blob/d88a5b7b59948d23977942ee62037e8912ff68ce/pyguide.md#312-todo-comments)

Reply:
Sorted :)

Reply:
Please wrap the TODO to 80 characeters